source/concepts/base-protocol.rst

```diff
@@ -11,23 +11,23 @@ The NanoVer protocol
 General architecture
 --------------------
 
-NanoVer provides three services: the state service, the trajectory service,
-and the command service. While none of these services are strictly
+NanoVer provides three services: the **state service**, the **trajectory service**,
+and the **command service**. While none of these services are strictly
 mandatory, some features expect two or three services to cooperate.
 
-The *state service* maintains a state that is shared between the server
+The **state service** maintains a state that is shared between the server
```

Comment:
In all cases "maintains a state" should probably be something clearer along the lines of "coordinates a shared data store"--just "state" alone is probably too vague.

Reply:
A good suggestion, I've updated the file to use your suggested wording for all instances of "maintains a state"